x-pack/plugins/observability/public/pages/slos/components/badges/slo_indicator_type_badge.tsx

```diff
@@ -45,11 +45,15 @@ export function SloIndicatorTypeBadge({ slo, color }: Props) {
       <EuiFlexItem grow={false}>
         <EuiBadge
           color={color ?? euiLightVars.euiColorDisabled}
-          onClick={() => {
+          onClick={(evt) => {
+            evt.stopPropagation();
```

Comment:
I see you stop the proPagation here, but looks what happens in the SLO list page. Clicking on the badge, refreshes the page. Maybe just left it be clickable and lead to SLO detail page? 

https://github.com/elastic/kibana/assets/2852703/bcd422a6-d1d4-48a4-a7db-361edf3d3950

Reply:
Interesting, clicking first time on the badge refreshes the page, but if I click again on the badge it doesn't do anything.